app/presenters/formats/edition_format_presenter.rb

```diff
@@ -56,7 +46,24 @@ def document_type
     end
 
     def details
-      {}
+      { phase_message: phase_message }.delete_if { |_key, value| value.blank? }
+    end
+
+    def phase_message
+      if show_phase_message?
+        {
+          content: edition.phase_message,
+          content_type: "text/html",
```

Comment:
It looks like we're changing the format of the payload here, so it now includes a content type. Can you explain this in the commit message? (It could also be its own commit.)